api/openapi-spec/swagger.json

```diff
@@ -3005,6 +3005,19 @@
       }
      }
     }
+   },
+   "/apis/subresources.kubevirt.io/v1alpha1/version": {
```

Comment:
why not just "/version" ?

Reply:
Because if we want to use only version we need to register the api server again without the v1alpha1 and i dont thinks its critical.

If its a problem I can add a new ApiregistrationV1beta1 just tell me.